src/script/conversation/ConversationRepository.ts

```diff
@@ -1968,6 +1993,10 @@ export class ConversationRepository {
       }
     }
 
+    if (type === CONVERSATION_EVENT.MEMBER_LEAVE) {
+      this.handleWipeMLSConversation(conversationId.id);
+    }
+
```

Comment:
I think this code belongs in the `reactToConversationsEvent` and there you won't event have to fetch the conversation, the code before it will have done it for you

Reply:
The thing is this function won't even get called (we get 403 access denied error)